releasenotes/v1.11.0.md

```diff
@@ -0,0 +1,54 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.10.0
+
+## :warning: Breaking Changes
+
+- Remove webhooks from api folder (#1042)
+
+## :bug: Bug Fixes
+
+- Fix infinite reconciles and timestamp updates (#1087)
+- Fix set-manifest-image rule (#1056)
+- Error handling for existing claims with an Error (#1005)
+
+## :book: Documentation
+
+- update build badges for release-1.11 (#1151)
+- update docs/releasing.md on dependabot configs (#1064)
+
+## :seedling: Others
+
+- Bump golang to v1.24.7 (#1150)
+- Bump github.com/onsi/ginkgo/v2 from 2.25.2 to 2.25.3 in /api (#1147)
+- Bump github.com/onsi/ginkgo/v2 from 2.25.2 to 2.25.3 (#1146)
+- Bump the kubernetes group to v0.33.5 (#1141)
+- Bump github.com/spf13/pflag from 1.0.9 to 1.0.10 (#1144)
+- Bump CAPI to v1.11.1 (#1125)
+- Bump github.com/onsi/gomega to v1.38.2 (#1118)
+- Bump github.com/onsi/gomega from 1.38.1 to 1.38.2 (#1107)
+- Enable GCI linter (#1102)
+- fix osv-scanners Slack reporting (#1101)
+- Add continue-on-error and improve Slack notifications for osv-scanner (#1095)
+- Bump OSV scanner action to fix SARIF upload issue (#1094)
+- Bump osv-scanner to v2.1.0 (#1088)
```

Comment:
This is dupe of #1094, which bumps it to v2.2.0

Reply:
Good catch, thanks, fixed.